pkg/webservice/handlers.go

```diff
@@ -339,3 +342,55 @@ func getClusterConfig(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 	}
 }
+
+var mutex sync.Mutex
+func updateConfig(w http.ResponseWriter, r *http.Request) {
+	mutex.Lock()
+	defer mutex.Unlock()
+	writeHeaders(w)
+	requestBytes, err := ioutil.ReadAll(r.Body)
+
+	if err == nil {
+		// validation is already called when loading the config
+		schedulerConf, err2 := configs.LoadSchedulerConfigFromByteArray(requestBytes)
```

Comment:
Can we re-use err instead of having multiple ones err2, err3, and err4?

Reply:
Done